src/Starlight/Launch/Launcher.cs

```diff
@@ -98,11 +86,11 @@ public static ClientInstance Launch(LaunchParams info, Client client)
 
         // ReSharper disable once MethodSupportsCancellation
         windowTask.Wait();
-
+        
         if (cancelSrc.IsCancellationRequested)
```

Comment:
Before I crap myself trying to figure this out any time the `cancelSrc` in this function is cancelled, Roblox prematurely closed.